homeassistant/components/ezviz/entity.py

```diff
@@ -3,6 +3,7 @@
 
 from typing import Any
 
+from homeassistant.helpers import device_registry
```

Comment:
There is no reason to import the whole module at this point.

```suggestion
from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC 
```